portfolio.css

```diff
@@ -84,6 +88,7 @@ nav{
 .home:hover{
 		border-bottom: 2px solid;
```

Comment:
Maybe text-decoration:underline instead?

Reply:
Was looking for a shadow type effect and not the underlined one in this

Reply:
Was looking for a shadow type effect and not the underlined one in this